src/utils/tests-utils/mock-db.js

```diff
@@ -18,8 +19,8 @@ class MockDB {
     });
   };
 
-  createUser = (username, password, role = permissions.USER.value) => {
-    const createdUser = this.DAO.create(username, password, role);
+  createUser = async (username, password, role = permissions.USER.value) => {
+    const createdUser = await this.DAO.create(username, password, role);
```

Comment:
why do you need an intermediary variable?
just do `return this.DAO.create()` it can be even done implicitely